### PR TITLE
add channels.name autocomplete test

### PR DESCRIPTION
**Pull request by imtoori**
add test for https://github.com/GetStream/chat/pull/1077

### CODE REVIEW

File: test/channels.js [hunk 1]
```diff
@@ -1566,6 +1566,35 @@ describe('$ne operator', function() {
...
+		});
+		await channel.create();
+	});
+
+	it('return 1 result', async function() {
+		console.log(channel.data.name);
+		console.log(channel.data.name.substring(0, 3));
+		const resp = await client
+			.queryChannels({
+				name: {
```
Comment (thesyncim) [inline test/channels.js:1588]:
```suggestion
				name: {
				members: [user],
```

Comment (thesyncim) [reply test/channels.js:1588]: better to be a bit more selective

Comment (ferhatelmas) [reply test/channels.js]: better, indeed @imtoori

Comment (ferhatelmas) [reply test/channels.js]: I pushed this change, merging.

**Pull request merged**